### PR TITLE
add verify_email file path to emailOption in register

### PR DESCRIPTION
**Pull request by debeemedia**
@promise-J

### CODE REVIEW

File: views/verify_email.ejs [hunk 1]
```diff
@@ -82,7 +82,7 @@
         <p class="email-message">
           You are welcome to Aphia! We are glad to have you onboard. Kindly
           click on this link to verify your account
-          <a href="https://aphia.onrender.com/verify/?email=<%=email %>">Click here</a>
+          <a href="https://aphia.onrender.com/api/verify/?email=<%=email %>">Click here</a>
```
Comment (promise-J) [inline views/verify_email.ejs]: Make changes to this to reflect the front end URL unless you have a get method that handles this route

**Review by promise-J — CHANGES_REQUESTED** [review]

File: views/verify_email.ejs [hunk 2]
```diff
@@ -82,7 +82,7 @@
         <p class="email-message">
           You are welcome to Aphia! We are glad to have you onboard. Kindly
           click on this link to verify your account
-          <a href="https://aphia.onrender.com/api/verify/?email=<%=email %>">Click here</a>
+          <a href="https://blackholemain.onrender.com/api/verify/?email=<%=email %>">Click here</a>
```
Comment (promise-J) [inline views/verify_email.ejs:85]: Same here. point to the front end url

**Review by promise-J — CHANGES_REQUESTED** [review]

File: views/verify_email.ejs (hunk 1 above)
Comment (debeemedia) [reply views/verify_email.ejs]: we need to test it from the backend first

File: views/verify_email.ejs (hunk 2 above)
Comment (debeemedia) [reply views/verify_email.ejs:85]: we need to test it from the backend first

**Pull request merged**